File: 0058-length-of-last-word/0058-length-of-last-word.cs
public class Solution {
    public int LengthOfLastWord(string s) {
        s=s.Trim();
        int n=s.Length;
        int cnt=0;
        for(int i=n-1;i>=0;i--){
            if(s[i]!=' '){
                cnt++;
            }
            else{
                break;
            }
        }
        return cnt;

    }
}

File: 0058-length-of-last-word/0058-length-of-last-word.java
class Solution {
    public int lengthOfLastWord(String s) {
        s=s.strip();
        String[] st=s.split(" ");
        int n=st.length;
        int maxs=0;
        int len=0;
        for(int i=0;i<n;i++){
            String a=st[i];
            len=a.length();
            if(len>maxs){
                maxs=len;
            }
        }
        return len;
    }
}